lettuce/flows/__init__.py

```diff
@@ -4,3 +4,11 @@
 
 from lettuce.flows.taylorgreen import TaylorGreenVortex2D, TaylorGreenVortex3D
 from lettuce.flows.couette import CouetteFlow2D
+from lettuce.flows.channel import ChannelFlow2D
+
+def flow_by_name(flow_class=None):
+    FlowDictorary = {
+        "TGV2D": TaylorGreenVortex2D,
+        "TCF2D": ChannelFlow2D,
+        "COU2D": CouetteFlow2D}
+    return FlowDictorary if flow_class is None else FlowDictorary[flow_class]
```

Comment:
You need this only once (also defined in cli.py)

Also:
I don't mind making this a function, but why not just a dictionary? 

```
flow_by_name = { ... }
```

Minor:
TCF2D -> I would prefer Channel2D
COU2D -> I would prefere Couette2D
(TGV2D -> Maybe TaylorGreen2D ?)
I know, I know: But in this way we don't need to write documentation for these.